src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -28,32 +31,60 @@
     private $securityService;
 
     /**
+     * @var Session
+     */
+    private $minkSession;
+
+    /**
+     * @var array
+     */
+    private $minkParameters;
+
+    /**
      * @var HomePage
      */
     private $homePage;
 
     /**
-     * @var Session
+     * @var TestUserFactoryInterface
      */
-    private $minkSession;
+    private $testUserFactory;
 
     /**
-     * @var array
+     * @var RepositoryInterface
```

Comment:
In fact, why? ;) the only benefit, which we have from using ``UserRepositoryInterface`` is ``findOneByEmail`` method, which is not used in this class.